pxf/build.gradle

```diff
@@ -434,7 +435,6 @@ task tomcatRpm(type: Rpm) {
         packageName 'apache-tomcat'
         summary = 'Apache Tomcat RPM'
         vendor = project.vendor
-        release = buildNumber()
```

Comment:
The HAWQ pulse build number is not relevant for the apache tomcat rpm.
Besides we are not releasing internal custom builds of apache tomcat